### PR TITLE
TC1001 Login test

### PR DESCRIPTION
**Pull request by Ozleblebici**
Local test completed, ready for review 

**Ticket :** www.jira.com/TC1001

**Local Test result;**
image

### CODE REVIEW

File: src/Test1.java [hunk 1]
```diff
@@ -1,10 +1,10 @@
 public class Test1 {
 
     public static void main(String[] args) {
-        System.out.println("local test 1");
```
Comment (TalhaOzleblebici) [inline src/Test1.java:4]: Why did you remove existing test1?

Comment (Ozleblebici) [reply src/Test1.java:4]: ah sorry missed this one let me roll back

Comment (Ozleblebici) [reply src/Test1.java:4]: @TalhaOzleblebici Updated

Comment (TalhaOzleblebici) [reply src/Test1.java:4]: Thank you

**Review by TalhaOzleblebici — APPROVED** [review]

**Comment by CevahirBb** [comment]
Can you add one more line test2?

**Review by RosanaKmetovikj — APPROVED** [review]
